<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>313</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>721</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Enter text for analysis here. Click'Analyse text (Alt+y)' for multiple tex analysis.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ReportButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProcessButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Process Below Text</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLinguistic_Analysis">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuText_Analysis">
    <property name="title">
     <string>Text Analysis</string>
    </property>
    <addaction name="actionAnalyse_text"/>
    <addaction name="actionReport"/>
   </widget>
   <addaction name="menuLinguistic_Analysis"/>
   <addaction name="menuText_Analysis"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAnalyse_text">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Analyse text</string>
   </property>
   <property name="toolTip">
    <string>Submits text for analysis </string>
   </property>
   <property name="whatsThis">
    <string>Submits text for analysis </string>
   </property>
   <property name="shortcut">
    <string>Alt+Y</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Report</string>
   </property>
   <property name="toolTip">
    <string>Analyse avaliable text and displaies a report</string>
   </property>
   <property name="whatsThis">
    <string>Analyse avaliable text and displaies a report</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clears the database and imput.</string>
   </property>
   <property name="whatsThis">
    <string>Clears the database and imput.</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
